plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -219,15 +219,15 @@ SecRule REQUEST_FILENAME "@contains /apps/recommendations/settings/enabled" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
-# Text app autosave sync feature doesn't works 
+# Text app autosave sync feature doesn't work 
 SecRule REQUEST_URI "@contains /apps/text/session/sync" \
     "id:9508126,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
-    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:autosaveContent"
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS"
```

Comment:
Can we be more restrictive here? `autosaveContent` seems to have been used once so we definitely need to keep it. What's the new name of the argument?

It would also help to document the versions the different argument names apply to.